BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -94,37 +94,37 @@ + (NSString *)iconNameForFileExtension:(NSString *)fileExtension
 {
     NSString *iconName = nil;
     if ([[self audioFileExtensions] containsObject:fileExtension])  {
-        iconName = @"Audio";
+        iconName = @"music";
```

Comment:
minor, I kinda like these as 'audio' and ' image' as opposed to 'music' and 'photo', at least on our side. but ok